packages/iTwinUI-react/src/core/Header/HeaderBreadcrumbsSplitButton.tsx

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { ButtonProps } from '../Buttons/Button';
+import { IconButton } from '../Buttons/IconButton';
+import { DropdownMenu } from '../DropdownMenu';
+import { Placement } from 'tippy.js';
+import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
+import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
+
+import { PolymorphicForwardRefComponent, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/button.css';
+
+export type HeaderBreadcrumbsSplitButtonProps = ButtonProps & {
+  /**
+   * Items in the dropdown menu.
+   * Pass a function that takes the `close` argument (to close the menu),
+   * and returns a list of `MenuItem` components.
+   */
+  menuItems: (close: () => void) => JSX.Element[];
+  /**
+   * Placement of the dropdown menu.
+   * @default 'bottom-end'
+   */
+  menuPlacement?: Placement;
+  /**
+   * Content of primary button.
+   */
+  children: React.ReactNode;
+};
+
+type HeaderBreadcrumbsSplitButtonComponent = PolymorphicForwardRefComponent<
+  'button',
+  HeaderBreadcrumbsSplitButtonProps
+>;
+
+/**
+ * Header Breadcrumb button component with a DropdownMenu.
+ *
+ */
+export const HeaderBreadcrumbsSplitButton: HeaderBreadcrumbsSplitButtonComponent = React.forwardRef(
```

Comment:
The name can be changed

The reason for the Dropdown being a separate component than the button is because I was having issues returning both the `Button` and the `DropdownMenu` in one component unless they were wrapped together in another element. I could wrap them both in an `<li>` element to follow the DOM structure Jon has in CSS, but I was going to apply that wrapper in `HeaderButton.tsx` or in a new `BreadcrumbButton` component.